8-code-editor/1-using-a-code-editor/README.md

```diff
@@ -153,9 +153,9 @@ After installing and using your extension, vscode.dev offers options to manage y
 
 <!----
 ## Post-Lecture Quiz
-[Post-lecture quiz](https://ashy-river-0debb7803.1.azurestaticapps.net/quiz/4)
+[Post-lecture quiz](https://ashy-river-0debb7803.1.azurestaticapps.net/quiz/8)

```

Comment:
This is commented out as a reminder to create a post-lecture quiz. Neither link is correct to this lessos, so it can be left as is.